Sources/PrimerSDK/Classes/User Interface/Root/PrimerNavigationBar.swift

```diff
@@ -25,7 +25,7 @@ class PrimerNavigationBar: PrimerView {
     
     var rightBarButton: UIButton? {
         didSet {
-            rightBarButton?.tintColor = theme.text.system.color
+            rightBarButton?.tintColor = .systemRed
```

Comment:
why not our red error color?

Reply:
thx, good catch! I'll fix this and do an extra scan for these